app/views/provider_interface/provider_relationship_permissions/edit.html.erb

```diff
@@ -35,20 +38,24 @@
 
       <div class="govuk-form-group">
         <%= f.fields_for :training_provider_permissions, @form.training_provider_permissions do |tpp| %>
-          <%= tpp.govuk_check_boxes_fieldset :permissions, legend: {
+          <%= tpp.govuk_check_boxes_fieldset :permissions, classes: 'training_provider', legend: {
```

Comment:
Please could the `_` be a `-` (here and in `accredited_body`) to match convention elsewhere?